txp/Pos/Txp/Logic/Global.hs

```diff
@@ -70,7 +72,7 @@ verifyBlocks ::
 verifyBlocks verifyAllIsKnown newChain = runExceptT $ do
     bvd <- gsAdoptedBVData
     let verifyPure :: [TxAux] -> UtxoM (Either ToilVerFailure TxpUndo)
-        verifyPure = runExceptT . verifyToil bvd epoch verifyAllIsKnown
+        verifyPure = runExceptT . verifyToil bvd (tcBlacklistSrcAddrs given) epoch verifyAllIsKnown
```

Comment:
is there an alias for `given` for this case?

Reply:
Yes, there is. Will use it.